### DISCUSSION TITLE
Number of valves survey

### DISCUSSION THREAD

**Discussion by nliaudat** [nliaudat/esp32_8ch_motor_shield#65] [Polls]
How many valves do you need?

**Comment by nstadigs** [comment]
13. I use two boards

Edit: To clarify, I have 13 valves in a single valve distributor

**Reply by marcelveldt** [reply]
16 (2x8) - I have 2 floor heating valve distributors (one upstairs, one downstairs) - So basically 2 boards (linked together)
As long as the boards are relative cheap, I think its fine that you can just chain them but maybe a minimum amount of valves so most people would have enough for a standard (single floor) distributor - so 8 I think ?

**Comment by DerMo92** [comment]
I have one central point per floor. So it is sufficient to have 6 valves since I use one PCB per floor. More difficult is if two circuits are on one bemf but one valve is quicker for some reason - both stop. So having 1:1 bemf could be a solution. Right now I manually shift them in time.

**Reply by nliaudat** [reply]
Sorry, I do not understand "having 1:1 bemf". 
What do you want in step or hardware or software modifications `?
Regards

**Reply by DerMo92** [reply]
Sorry if this was not clear for you. 
I just try to describe my issue and some suggestions how to solve. 
I solved this temporarily that two valves are running on one sensor by moving them time shifted even when triggered to move at the same time. 
Example: I have e.g two valves for kids room. If on top Temperatur both valves need to close. If on same sensor both stopp if one triggers the sensor even if not both are closed both stopp. 
So one to one sensor could also solve this but more sensors maybe also need more requirements I can't judge right now. 
So This is my situation maybe you have some thoughts about it. 
Thank you in advance 
BR
